server/integration/endpoint/src/test/resources/org/infinispan/server/endpoint/endpoint-9.0.xml

```diff
@@ -18,14 +18,34 @@
         </sasl>
       </authentication>
    </hotrod-connector>
-    <hotrod-connector name="hotrod5" socket-binding="hotrod" cache-container="default">
-        <encryption security-realm="other">
-            <sni host-name="sni" security-realm="other" />
-            <sni host-name="sni2" security-realm="other2" />
-            <sni host-name="sni3" />
-        </encryption>
-    </hotrod-connector>
+   <hotrod-connector name="hotrod5" socket-binding="hotrod" cache-container="default">
+      <encryption security-realm="other">
```

Comment:
I'm not sure it deserves a comment... it was just reformatting (I highly recommend reviewing with ignoring whitespaces changes - this way you won't see this kind of things).